### PR TITLE
refactor: Added generic loader [140]

### PR DESCRIPTION
**Pull request by Astitva877**
## Description

Please do not leave this blank 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/lib/components/Transition/loader/LoaderV2.svelte [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    justify-content: center;
+    align-items: center;
+  }
+
+  .loading-spinner {
+    width: 100%;
+    height: 100%;
+    -webkit-backdrop-filter: blur(3px);
+    backdrop-filter: blur(3px);
+    z-index: 999;
```
Comment (LordNayan) [inline src/lib/components/Transition/loader/LoaderV2.svelte:35]: make it 1000

File: src/lib/components/collections/req-res-section/sub-components/request-body-section/RequestParam.svelte [hunk 2]
```diff
@@ -29,6 +29,7 @@
   import type { CollectionsMethods } from "$lib/utils/interfaces/collections.interface";
   import type { NewTab } from "$lib/utils/interfaces/request.interface";
   import { Pane, Splitpanes } from "svelte-splitpanes";
+  import LoaderV2 from "$lib/components/Transition/loader/LoaderV2.svelte";
```
Comment (LordNayan) [inline src/lib/components/collections/req-res-section/sub-components/request-body-section/RequestParam.svelte:32]: Rename this to Loader.

File: src/lib/components/Transition/LoginWaitingScreen.svelte [hunk 3]
```diff
@@ -1,69 +1,63 @@
...
+
+<div
+  class="container d-flex flex-column mb-0 px-4 pb-0 pt-4"
+  transition:fly={{ y: 50, delay: 0, duration: 100 }}
+  on:introstart
+  on:outroend
+>
+  <div class="d-flex align-items-center justify-content-center mb-3 gap-2">
+    <img src={icons.logoSparrow} alt="" />
+    <h5 class="mb-0 text-whiteColor" style="font-weight: 500;font-size:36px;">
```
Comment (parthdalaltech) [inline src/lib/components/Transition/LoginWaitingScreen.svelte:26]: https://getbootstrap.com/docs/5.3/utilities/text/#font-weight-and-italics for font weight and font size can be set in rem

File: src/lib/components/Transition/LoginWaitingScreen.svelte [hunk 4]
```diff
@@ -1,69 +1,63 @@
...
+    height: 100vh;
+    background: black;
+    -webkit-backdrop-filter: blur(3px);
+    backdrop-filter: blur(3px);
+    z-index: 1;
+  }
+
+  .container {
+    position: fixed;
+    height: 200px;
```
Comment (parthdalaltech) [inline src/lib/components/Transition/LoginWaitingScreen.svelte:54]: cant we have div to take same height width with tansparent backgriund and z-index greater than overlay ? 
and elements aligned at center ?

File: src/lib/components/Transition/LoginWaitingScreen.svelte [hunk 5]
```diff
@@ -1,69 +1,63 @@
...
+    width: 500px;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: var(--background-color);
+    z-index: 1;
+    border-radius: 10px;
```
Comment (parthdalaltech) [inline src/lib/components/Transition/LoginWaitingScreen.svelte:61]: https://getbootstrap.com/docs/5.3/utilities/borders/#sizes for border radius

File: src/lib/components/Transition/LoginWaitingScreen.svelte [hunk 6]
```diff
@@ -1,69 +1,63 @@
...
+
+  .container {
+    position: fixed;
+    height: 200px;
+    width: 500px;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: var(--background-color);
+    z-index: 1;
```
Comment (parthdalaltech) [inline src/lib/components/Transition/LoginWaitingScreen.svelte:60]: https://getbootstrap.com/docs/5.3/utilities/z-index/#overlays for z-index

**Review by parthdalaltech — CHANGES_REQUESTED** [review]

File: src/lib/components/Transition/loader/LoaderV2.svelte (hunk 1 above)
Comment (Astitva877) [reply src/lib/components/Transition/loader/LoaderV2.svelte]: resolved

File: src/lib/components/collections/req-res-section/sub-components/request-body-section/RequestParam.svelte (hunk 2 above)
Comment (Astitva877) [reply src/lib/components/collections/req-res-section/sub-components/request-body-section/RequestParam.svelte]: resolved

File: src/lib/components/Transition/LoginWaitingScreen.svelte (hunk 3 above)
Comment (Astitva877) [reply src/lib/components/Transition/LoginWaitingScreen.svelte]: resolved

File: src/lib/components/Transition/LoginWaitingScreen.svelte (hunk 5 above)
Comment (Astitva877) [reply src/lib/components/Transition/LoginWaitingScreen.svelte]: resolved

**Review by parthdalaltech — APPROVED** [review]

**Pull request merged**